media-plugins/gst-plugins-webrtc/gst-plugins-webrtc-1.18.4-r1.ebuild

```diff
@@ -10,9 +10,22 @@ DESCRIPTION="WebRTC plugins for GStreamer"
 KEYWORDS="~amd64"
 IUSE=""
 
+# == ext/webrtc/meson.build
+# dev-libs/glib (eclass): gio_dep
+# net-libs/libnice: libnice_dep
+# media-libs/gst-plugins-base: gstbase_dep, gstsdp_dep, gstapp_dep, gstrtp_dep
+# media-plugins/gst-plugins-sctp: gstsctp_dep
+# == ext/webrtcdsp/meson.build
+# media-libs/gst-plugins-base: gstbase_dep, gstaudio_dep
+# media-libs/gst-plugins-bad: gstbadaudio_dep
+# media-libs/webrtc-audio-processing: webrtc_dep
+# (android): gnustl_dep
+# == gst-libs/gst/webrtc/meson.build
+# media-libs/gst-plugins-base: gstbase_dep, gstsdp_dep
```

Comment:
What's this about?

Reply:
Correspondence between our packages and dependency declarations used for the webrtc meson.build files.